### ISSUE TITLE
Problem: broken navigation (the mobile navigation isn't implemented by design)

### ISSUE THREAD

**Issue by predrag-us** [holdex/holdex-venture-studio#405]
### [Figma Design ⏤ Preview](https://www.figma.com/file/39MGGVIGQ1bL3FgVzuXEdC/UI-Design?type=design&node-id=569%3A8286&t=4a33CsnMc4AMca83-1)

---

Problem: The mobile navigation isn't implemented by design. 

Please see the design while solving the following problems and mind the notes if any.

```[tasklist]
### Problems
- [ ]  Can't tap the logo to go to `/`. The fourth point should solve this since the fade is misplaced. 
- [ ] `Partner With Us` should be in the form of a detached button located at the bottom-right of the screen.
- [ ] `themeSwitcher` shouldn't be always visible, but a part of the scrollable navigation.
- [ ] `navItem(s)` shouldn't go below the logo ⏤ they should fade right after the logo.
- [ ] `scrollBar` shouldn't be visible when scrolling.
- [ ] The color of the `lightTheme` fade isn't according to the design.
- [ ] The background color of the `lightTheme` navigation isn't according to the design ⏤ this problem is valid on all screens.
```

**Comment by winter-sols** [comment]
@predrag-us , @georgeciubotaru , @markholdex I'm starting this issue.

**Comment by predrag-us** [comment]
@winter-sols What's your status? Can you please provide an ETA and attach your PR to the issue? Thanks!

**Comment by winter-sols** [comment]
Now, I'm focusing on the issue-https://github.com/holdex/holdex-venture-studio/issues/446
I will do this after I finish current #446.

**Comment by predrag-us** [comment]
@winter-sols 

> @predrag-us , @georgeciubotaru , @markholdex I'm starting this issue.

Why this message, then? 

--- 

I'm sensing confusion here. 

This issue is about fixing the current navigation, while #446 is about implementing the new navigation.

Therefore, once you complete #446, this issue can be safely closed since the new navigation is already implemented and we don't need to fix the current one. Makes sense? 

Also, just a friendly reminder. Mark let you know that #446 is `Low` priority while this one is `Urgent`. 

We execute on `High` and `Urgent` issues first. 

You're free to pick the issue you want to work on, just be aware of the priority.

**Comment by predrag-us** [comment]
Solved at #446 ⏤ closing the issue.
